meta-balena-common/recipes-support/balena-config-vars/balena-config-vars/balena-config-vars

```diff
@@ -38,6 +38,28 @@ done
 BOOT_MOUNTPOINT="/mnt/boot"
 FLASHER_FILEFLAG="$BOOT_MOUNTPOINT/balena-image-flasher"
 
+fatcp() {
+    /usr/bin/fatrw copy "${1}" "${2}"
+}
+
+fatmv() {
+    fatcp "${1}" "${2}"
+    rm -rf "${1}"
+}
+
+fatcat() {
+    /usr/bin/fatrw read "${1}"
+}
+
+CP="/bin/cp"
+MV="/bin/mv"
+CAT="/bin/cat"
+if command -v "/usr/bin/fatrw" > /dev/null; then
+    CP=fatcp
+    MV=fatmv
+    CAT=fatcat
+fi
```

Comment:
Why do we need to fallback to the unsafe commands? Do we have a case where fatrw will not be available? My worry is that here we are doing a silent fallback to the unsafe commands and if for some reason fatrw is not found that may lead to unwanted behavior.

Reply:
Right, I'll remove the fallbacks and just check for the existence of `fatrw` and exit otherwise.